crates/ruff/src/rules/flake8_comprehensions/rules/unnecessary_double_cast_or_process.rs

```diff
@@ -31,6 +31,7 @@ use super::helpers;
 ///
 /// This rule applies to a variety of functions, including `list`, `reversed`,
 /// `set`, `sorted`, and `tuple`. For example:
+///
```

Comment:
This is done to avoid grouping all the list elements into a single line: https://beta.ruff.rs/docs/rules/unnecessary-double-cast-or-process/

I generated the docs locally and confirmed that the elements are now rendered as required.